hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenRevokeKycFromAccountHandler.java

```diff
@@ -60,7 +63,7 @@ public void preHandle(@NonNull final PreHandleContext context, @NonNull final Re
         requireNonNull(context);
         final var op = context.body().tokenRevokeKycOrThrow();
         final var tokenMeta = tokenStore.getTokenMeta(op.tokenOrElse(TokenID.DEFAULT));
-        if (tokenMeta == null) throw new PreCheckException(INVALID_TOKEN_ID);
+        if (tokenMeta == null) throw new PreCheckException(ResponseCodeEnum.INVALID_TOKEN_ID);
         if (tokenMeta.hasKycKey()) {
```

Comment:
⚠️ Don't we need to fail here with `TOKEN_HAS_NO_KYC_KEY` if the KYC key is missing?

Reply:
I only added the class name, but I can change this too 👍